setupsrc/pypdfium2_setup/update.py

```diff
@@ -178,12 +178,21 @@ def do_verify(archives, pdfium_version, have_recent_gh):
     
     verify_result = run_cmd(
         ["gh", "attestation", "verify", "-R", "bblanchon/pdfium-binaries", str(one_artifact), "-b", str(attest_path), "--format=json"],
-        cwd=DataDir, check=True, capture=True
+        cwd=DataDir, check=True, capture=True, stderr=subprocess.PIPE
     )
     log(f"{one_artifact.name}: verified by gh")
     
-    verify_result = json.loads(verify_result)
-    attested_artifacts = verify_result[0]["verificationResult"]["statement"]["subject"]
+    # Handle potential empty or malformed JSON from gh attestation verify
+    if verify_result.strip():
+        verify_result = json.loads(verify_result)
+        # gh attestation verify returns a list of verification results
+        if isinstance(verify_result, list) and len(verify_result) > 0:
+            verify_result = verify_result[0]
+    else:
+        # Skip verification if output is empty
+        log(f"{one_artifact.name}: skipping verification due to empty output")
+        return
```

Comment:
If this scenario happens
* an error would have to be raised if `--verify` was explicitly requested
* this is a serious issue that would need to be fixed properly in any case, e.g. by switching back to the previous one-by-one verification if need be.